indico_toolkit/pipelines/file_processing.py

```diff
@@ -45,6 +46,27 @@ def get_file_paths_from_dir(
             f"Found {len(self.file_paths)} valid files and {len(self.invalid_suffix_paths)} paths with invalid suffixes."
         )
 
+    def move_all_file_paths(
+        self, 
+        origin_dir,
+        destination_dir,
+        accepted_types: Tuple[str],
+        copy_files: bool = False,
+    ):
+        self.get_file_paths_from_dir(origin_dir, accepted_types, True)
+        if os.path.isdir(destination_dir):
+            if destination_dir[-1] != '/':
+                destination_dir = f'{destination_dir}/'
+            for initial_filepath in self.file_paths:
+                file_to_be_moved = os.path.basename(initial_filepath)
+                if copy_files == False:
+                    os.rename(initial_filepath, f'{destination_dir}{file_to_be_moved}')
+                else: 
+                    shutil.copyfile(initial_filepath, f'{destination_dir}{file_to_be_moved}') 
```

Comment:
you should not construct the file path in the function call. you shoul dalso be using pathlib to do something more clean like `output_filepath = destination_dir / file_to_be_moved`